ovs/extensions/healthcheck/arakoon/arakooncluster_health_check.py

```diff
@@ -381,80 +341,60 @@ def verify_integrity(logger, arakoon_clusters=None):
         # Processing results
         ver_result = working_arakoon_list, no_master_arakoon_list, down_arakoon_list, unkown_arakoon_list
         if len(ver_result[0]) == len(arakoon_clusters):
```

Comment:
Hmmm.... Nice example of making code more unreadable :)
All the variables are there, but lets put them in an extra tuple to return the same variables again in the end of the function instead of the (uselessly) created tuple